state/indexer/block/kv/kv_test.go

```diff
@@ -358,3 +358,132 @@ func TestBlockIndexerMulti(t *testing.T) {
 		})
```

Comment:
There were some test cases removed from the this test in the original patch. Any reason for them to stay here?

Reply:
The reason they are here is because in 0.34 we have two tests for the same query - one with `match.events`, and one without. In 0.37 they were left over and hence deleted. Here we need both.